teitelbot/response/quote/__init__.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from typing import Callable, Any
+
+from response import classifier
+from response.action import Action
+from response.exception import ResponseNotExistForAction
+
+LOGGER = logging.getLogger(__name__)
+
+
+def on_message(text: str, reply: Callable[[str], Any]) -> None:
+    actions = classifier.possible_classifications(text)
+    for action in actions:
```

Comment:
If there are several possible classifications, will teitelbot just spam a bunch of responses?

Reply:
The break is what stops it?
Is it normal in python to throw an exception even if the flow is good?

Reply:
1. no. only the first response.
2. idk